src/js/claims-status/components/appeals-v2/AppealsV2TabNav.jsx

```diff
@@ -3,12 +3,15 @@ import PropTypes from 'prop-types';
 
 import TabItem from '../TabItem';
 
-const AppealsV2TabNav = ({ appealId }) => {
+const AppealsV2TabNav = ({ appealId, claimHeading }) => {
   return (
-    <ul className="va-tabs claims-status-tabs small-12 medium-10" role="tablist">
-      <TabItem shortcut={1} className="appeals-tabs-item" id="v2status" tabpath={`appeals-v2/${appealId}/status`} title="Status"/>
-      <TabItem shortcut={2} className="appeals-tabs-item" id="v2detail" tabpath={`appeals-v2/${appealId}/detail`} title="Detail"/>
-    </ul>
+    <div>
+      <h1>{claimHeading}</h1>
```

Comment:
Just for separation of concerns, can we pull this header out and put it directly in `AppealInfo`?

Reply:
That seems like a very reasonable idea.

Reply:
Also happens to introduce a visual regression in the help sidebar top alignment; I'll open a new issue for that.